index.html

```diff
@@ -49,8 +49,17 @@
   </head>
 
   <body>
-    <h1 id="header">Basics: Beat That! 🚀</h1>
+    <h1 id="header">🤷‍♀️Basics: Can You Beat That?🤷‍♂️</h1>
```

Comment:
I love the colors and the use of emoji! These little touches make a lot of difference esp when im reviewing so many similar projects